responsibleai_vision/tests/test_feature_extractors.py

```diff
@@ -13,10 +13,22 @@
 
 MEAN_PIXEL_VALUE = ExtractedFeatures.MEAN_PIXEL_VALUE.value
 FRIDGE_METADATA_FEATURES = [
-    'Make', 'ResolutionUnit', 'ImageLength', 'ExifOffset', 'Model',
-    'GPSInfo', 'ImageWidth', 'DateTime', 'YCbCrPositioning',
-    'Software', 'Orientation'
-]
+    'SensingMethod', 'GPSVersionID', 'ISOSpeedRatings', 'SceneType',
+    'SceneCaptureType', 'SubjectDistance', 'CustomRendered',
+    'SubjectDistanceRange', 'DigitalZoomRatio', 'ApertureValue',
+    'ImageWidth', 'GPSDOP', 'MaxApertureValue', 'ColorSpace',
+    'FocalLengthIn35mmFilm', 'ExposureMode', 'Saturation', 'ExposureTime',
+    'ExifImageHeight', 'FNumber', 'YCbCrPositioning', 'Make', 'MeteringMode',
```

Comment:
great to see all of this additional metadata info retrieved now - it would be great to eventually also add a test with the images from the customer as well since they sent us those object detection images directly and said that we could use them publicly in the test cases

Reply:
Yep, planning for the next PR.